src/storage/conversion/TypedRepresentationConverter.ts

```diff
@@ -7,25 +8,22 @@ import type { RepresentationConverterArgs } from './RepresentationConverter';
  */
 export abstract class TypedRepresentationConverter extends RepresentationConverter {
   /**
-   * Get a hash of all supported input content types for this converter, mapped to a numerical priority.
-   * The priority weight goes from 0 up to 1.
```

Comment:
FYI, this line was added after this comment https://github.com/solid/community-server/pull/101#discussion_r481090356

Reply:
Yes, but those docs are now on `ValuePreferences`.